### PR TITLE
Add more tests cases

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

File: src/main/kotlin/org/jetbrains/research/ml/coding/assistant/util/psi/PsiUtil.kt [hunk 1]
```diff
@@ -12,6 +12,7 @@ val PsiElement.isLeaf: Boolean
     get() = this.children.isEmpty()
 
 // TODO: we don't store "in" keyword for loops in labels. Is it ok?
+// TODO: it does not support async keyword now and type annotations
 val PsiElement.intermediateElementLabel: String
```
Comment (elena-lyulina) [inline src/main/kotlin/org/jetbrains/research/ml/coding/assistant/util/psi/PsiUtil.kt:16]: Maybe we should add some description of why these types of PsiElement are considered and others are not? Is this a complete list of PsiElements we need to check?

File: src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/in_0.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+for i in range(100):
```
Comment (elena-lyulina) [inline src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/in_0.py:1]: shouldn't this file be in a 'for' subdirectory?

File: src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/out_0.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
```
Comment (elena-lyulina) [inline src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/out_0.xml:1]: shouldn't this file be in a 'for' subdirectory?

**Review by elena-lyulina — CHANGES_REQUESTED** [review]
Thanks, you've done a huge work! 
However, I'm curious is it possible to prove somehow that you have considered all cases in tests? maybe we should have a doc with some links to the python documentation and a list of considered cases?

File: src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/in_0.py (hunk 2 above)
Comment (nbirillo) [reply src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/in_0.py:1]: It is `break` and `continue` test case...It is general for both loops. But I can do it for both loops if you want

File: src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/out_0.xml (hunk 3 above)
Comment (nbirillo) [reply src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/out_0.xml:1]: The same answer as the previous comment

**Review by elena-lyulina — APPROVED** [review]
Great, thanks!

**Pull request merged**